### DISCUSSION TITLE
Clicking on "Artifacts" doesn't do anything

### DISCUSSION THREAD

**Discussion by digitalghost-dev** [community/community#88643] [Mobile]
### Select Topic Area

Bug

### Body

When viewing Actions for a repository and click on the "Artifacts" section, the app tries to redirect somewhere but doesn't and goes back to the Action.

https://github.com/community/community/assets/86637723/51221541-09b4-4792-a338-e59111b21997

**Comment by eliperkins** [comment]
Hey @digitalghost-dev! Thanks for the heads up on this behavior. This certainly seems like a bug to me, as the "Artifacts" button should link to a place where you can view said artifacts. I can understand your frustration here.

I've created an issue on our internal tracking repo for this bug. I'll work with our engineering team to get this addressed and follow back up with you here. Thanks for your patience!

**Reply by digitalghost-dev** [reply]
Sounds good, thank you!
